utils/config.ts

```diff
@@ -81,6 +81,13 @@ export const IConfig = t.intersection([
     isProduction: t.boolean
     /* eslint-enable sort-keys */
   }),
+  t.interface({
```

Comment:
Why creating an anonymous interface instead of fill env variables directly into the previous interface?

Reply:
No reason, actually, I'll fix it.

Reply:
Done in [a80e8ac](https://github.com/pagopa/io-functions-service-messages/pull/14/commits/a80e8ac41a6102b1d343ce02df8b75a9cde279c5)